.github/workflows/production-release-readiness.yml

```
@@ -68,8 +68,23 @@ jobs:
         run: |
           set -euo pipefail
 
-          docker buildx imagetools inspect "ghcr.io/${GITHUB_REPOSITORY}-backend:${TARGET_SHA}" >/dev/null
-          docker buildx imagetools inspect "ghcr.io/${GITHUB_REPOSITORY}-frontend:${TARGET_SHA}" >/dev/null
+          verify_image_platforms() {
+            local image="$1"
+
+            docker buildx imagetools inspect --raw "$image" \
+              | python -c 'import json, sys
+index = json.load(sys.stdin)
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)